create_cluster.py

```diff
@@ -426,6 +431,7 @@ def launch_instance(region: str, ip: dict, ami: object, subnet_id: str,
             SubnetId=subnet_id,
             PrivateIpAddress=ip['PrivateIp'],
             BlockDeviceMappings=block_devices,
+            IamInstanceProfile=instance_profile_name,
```

Comment:
Ah, forget it, I've overlooked the assignment above.  Then I must say the variable name is quite confusing. ;)